content/documentation/glossary/networking/index.adoc

```diff
@@ -0,0 +1,67 @@
+---
+title: "Networking"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation/glossary"
+---
+
+= Kiali Networking Glossary
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Networking Glossary Content
+:keywords: Kiali Documentation Glossary
+:icons: font
+
+== Destination
+
+Read the <<Destination>> definition in link:https://istio.io/help/glossary/#destination[Istio Glossary]
+
+=== Destination Rule
+
+Read the <<Destination Rule>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#DestinationRule[Istio Destination Rule]
+
+
+== Endpoint
+
+A communication endpoint is a type of communication network node. It is an interface exposed by a communicating party or by a communication channel
+
+== Error Rate
+
+It's the percentage the errors in the traffic to a specific object in a <<Rate Interval>>
+
+== Gateway
+
+Read the <<Gateway>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Gateway[Istio Gateway]
+
+== Inbound Metrics
+
+Incoming Metrics to a <<Workload>>, <<Service>> or <<Application>>
+
+== Outbound Metrics
+
+Outgoing Metrics to a <<Workload>> or <<Service>>
+
+== Port
+
+Read the <<Port>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Port[Istio Port]
+
+== Rate Interval
+
+It's an amount of time. By Default in Kiali last 10 minutes.
+
+== Rule
+
+It's an object that manages external access to the services in a cluster, typically HTTP
+
+== Source
+
+Read the <<Source>> definition in link:https://istio.io/help/glossary/#source[Istio Glossary]
+
+=== Source Services
+
+It's the origin service where something is generated.
```

Comment:
I'm not sure about this.   When do we have a source service?   Requests are generated by source workloads for a destination service.

If we do keep this I think it should just be "Source Service".  Also the definition is so vague as to be not useful.  Perhaps "something is" should be replaced with "requests are".

Reply:
The same https://github.com/kiali/kiali-ui/blob/c3b14438c4b7d84aac0e30f566079b70c3ca6174/src/pages/ServiceDetails/ServiceInfo.tsx#L147

Reply:
There is no such thing as a "source service". Istio even removed that term from the telemetry labeling because it was not correct. There is no service that "generates" something.

Look at Istio's glossary for source (https://istio.io/help/glossary/#source) 

"Within the service mesh a source is typically a workload, but the source for ingress traffic may include other clients such as a browser or mobile app."

So a source isn't a service. Then look at the definition for service and you'll see it probably makes no sense to call something a "source service" - https://istio.io/help/glossary/#service

Reply:
There is in Kiali UI:
![image](https://user-images.githubusercontent.com/1662329/45042469-0ebc2280-b06b-11e8-9a00-6e10176aa7af.png)
Which is the term that @aljesusg wanted to define, I guess.

Reply:
Yes, This is a glossary ok Kiali.... or I thought it....If not we can add a link to istio when the user click in the glossary.

Reply:
I agree with mazz, I think it's a problem in our UI. For a given service we may have source workloads but not source services.

Reply:
So we should change it in the UI. I added all names/parameter of the UI in this glossary by that.

Reply:
Let's document the version of Kiali that we release.
Let's open a JIRA.
Let's discuss it there, and then change the feature that today is working.
Then, change the documentation.

Please, in that order.

Reply:
I just checked. Kiali is realy fetching source workloads, there's really no "service" here. It's just a label issue, the rest of the feature looks correct.

Reply:
Then I would be for simply removing the term, rather than saying something vague, confusing and incorrect.